src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -65,6 +69,31 @@ public Emitter(Compilation compilation, bool emitPreamble = true) : base(emitPre
             return Capture();
         }
 
+        /// <summary>
+        /// Returns the nullable annotation string to use in the code generation according to the first first parameter of
```

Comment:
Nit: "...the first first parameter..."

Reply:
I am duplicating it to ensure users will read it :-) just kidding. I'll fix it.